internal/csi-common/utils.go

```diff
@@ -320,3 +320,46 @@ func IsBlockMultiNode(caps []*csi.VolumeCapability) (bool, bool) {
 
 	return isBlock, isMultiNode
 }
+
+// IsMultiWriterBlockRWOFile validates the volume capability slice against the access modes and access type.
+// if the capability is of multi write the first return value will be set to true and if the request
+// is of type block, the second return value will be set to true. If it is of type RWO and file mode, the third return
+// will be set to true. If the access mode is reader only the last return value has been set to true.
+// Otherwise, false values are returned for all the return arguments.
+func IsMultiWriterBlockRWOFile(caps []*csi.VolumeCapability) (bool, bool, bool, bool) {
+	var multiWriter bool // this flag has been set and returned after validating multi writer caps regardless of
```

Comment:
due to all the comments, the formatting is getting rather difficult to follow. It might be better to stick to the commenting above variables, similar to what is done for functions and members of a struct.

```go
var (
    // multiWriter has been set and returned after validating multi writer caps regardless of
    // single or multi node access mode. The caps check is agnostic to whether it is a filesystem
    // or block mode volume
    multiWriter bool
                                                      // <--- NOTE THE EMPTY LINE FOR READABILTY!!!
    // block has been set and returned if the passed in capability is of block volume mode
    block bool
)
```

Reply:
Done. actually it was not needed as part of function split